packages/visual-editor/src/components/Dropzone/useComponentProps.ts

```diff
@@ -204,3 +218,15 @@ export const useComponentProps = ({
 
   return { componentProps, wrapperProps };
 };
+
+const addExtraDropzonePadding = (padding: string) =>
+  padding
+    .split(' ')
+    .map((value) => {
+      if (value.endsWith('px')) {
```

Comment:
Could do some future proofing using these potential values - unless we know it'll always be px `export const SpacingUnitRegexp = /px|cm|mm|in|pt|pc|em|ex|ch|rem|vw|vh|vmin|vmax|%$/;`

Reply:
This method came from my PR originally. I agree it would be nice to support other units of measurement here, but it also seems uncommon for padding to be implemented using some of these options.

This method is used for detecting when there is `< 4px` padding on components so that we can attempt to expand the padding on sides of components that have less padding and avoid decreasing the padding on sides that have more than 4px.  

To support other units, we'd need to convert the value from other units to pixels to check if the converted pixel value `< 4px` or something that would help determine when `4px` is less than the value 🤔

Reply:
TLDR: We could add a `convertToPx` method but probably as a follow-up feature.

Reply:
Ticket for this: https://contentful.atlassian.net/browse/ALT-543